sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -698,6 +698,8 @@ private[sql] class SQLConf extends Serializable with CatalystConf with Logging {
   override def orderByOrdinal: Boolean = getConf(ORDER_BY_ORDINAL)
 
   override def groupByOrdinal: Boolean = getConf(GROUP_BY_ORDINAL)
+
+  override def allowCartesianProduct: Boolean = getConf(CROSS_JOINS_ENABLED)
```

Comment:
The `crossJoinEnabled` above also references the same underlying conf and is being used at multiple places in the code. Can we unify the two?